src/sas/system/log.py

```diff
@@ -12,70 +12,59 @@
 Module that manages the global logging
 '''
 
+#BASE_LOGGER = 'sasview'
+TRACED_PACKAGES = ('sas', 'sasmodels', 'sasdata', 'bumps', 'periodictable')
```

Comment:
nitpick: Did you mean `TRACKED_PACKAGES`?

Reply:
I mean the packages for which the debug traces should be printed during development.